frontend_tests/zjsunit/namespace.js

```diff
@@ -22,6 +22,47 @@ const real_jquery_path = require.resolve("../zjsunit/real_jquery.js");
 let in_mid_render = false;
 let jquery_function;
 
+const TEMPLATE_PATH = "/static/templates/";
+
+function need_to_mock_template_error(filename) {
+    const i = filename.indexOf(TEMPLATE_PATH);
+
+    if (i < 0) {
+        throw new Error("programming error");
+    }
+
+    const fn = filename.slice(i + TEMPLATE_PATH.length);
```

Comment:
Maybe call this `template_path`?

Reply:
Fixed.  We should probably move toward capitalizing module constants at some point, or even module-level variables.